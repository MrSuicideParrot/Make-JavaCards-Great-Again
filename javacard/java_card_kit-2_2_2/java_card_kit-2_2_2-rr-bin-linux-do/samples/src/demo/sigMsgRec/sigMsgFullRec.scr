
//+
// Copyright 2005 Sun Microsystems, Inc. All rights reserved.
// SUN PROPRIETARY/CONFIDENTIAL. Use is subject to license terms.
//-

//+
// Workfile:@(#)Header.scr	1.9
// Version:1.9
// Date:01/04/06
//-

powerup;

// Select the installer applet
0x00 0xA4 0x04 0x00 0x09 0xa0 0x00 0x00 0x00 0x62 0x03 0x01 0x08 0x01 0x7F;
// 90 00 = SW_NO_ERROR

// Turn output off during CAP file download
echo "CAP file download section. Output suppressed.";

output off;

0x80 0xB0 0x00 0x00 0x00 0x7F;

// com/sun/javacard/samples/sigMsgRec/javacard/Header.cap
0x80 0xB2 0x01 0x00 0x00 0x7F;
0x80 0xB4 0x01 0x00 0x16 0x01 0x00 0x13 0xDE 0xCA 0xFF 0xED 0x01 0x02 0x04 0x00 0x01 0x09 0xA0 0x00 0x00 0x00 0x62 0x03 0x01 0x0C 0x0C 0x7F;
0x80 0xBC 0x01 0x00 0x00 0x7F;

// com/sun/javacard/samples/sigMsgRec/javacard/Directory.cap
0x80 0xB2 0x02 0x00 0x00 0x7F;
0x80 0xB4 0x02 0x00 0x20 0x02 0x00 0x1F 0x00 0x13 0x00 0x1F 0x00 0x0E 0x00 0x1F 0x00 0x7E 0x00 0x0C 0x02 0x24 0x00 0x96 0x00 0x63 0x00 0x00 0x01 0x05 0x00 0x06 0x00 0x03 0x00 0x83 0x03 0x7F;
0x80 0xB4 0x02 0x00 0x02 0x01 0x00 0x7F;
0x80 0xBC 0x02 0x00 0x00 0x7F;

// com/sun/javacard/samples/sigMsgRec/javacard/Import.cap
0x80 0xB2 0x04 0x00 0x00 0x7F;
0x80 0xB4 0x04 0x00 0x20 0x04 0x00 0x1F 0x03 0x03 0x01 0x07 0xA0 0x00 0x00 0x00 0x62 0x01 0x01 0x03 0x01 0x07 0xA0 0x00 0x00 0x00 0x62 0x01 0x02 0x00 0x01 0x07 0xA0 0x00 0x00 0x00 0x62 0x7F;
0x80 0xB4 0x04 0x00 0x02 0x00 0x01 0x7F;
0x80 0xBC 0x04 0x00 0x00 0x7F;

// com/sun/javacard/samples/sigMsgRec/javacard/Applet.cap
0x80 0xB2 0x03 0x00 0x00 0x7F;
0x80 0xB4 0x03 0x00 0x11 0x03 0x00 0x0E 0x01 0x0A 0xA0 0x00 0x00 0x00 0x62 0x03 0x01 0x0C 0x0C 0x01 0x00 0x95 0x7F;
0x80 0xBC 0x03 0x00 0x00 0x7F;

// com/sun/javacard/samples/sigMsgRec/javacard/Class.cap
0x80 0xB2 0x06 0x00 0x00 0x7F;
0x80 0xB4 0x06 0x00 0x0F 0x06 0x00 0x0C 0x00 0x80 0x03 0x06 0x00 0x04 0x07 0x01 0x00 0x00 0x00 0xA3 0x7F;
0x80 0xBC 0x06 0x00 0x00 0x7F;

// com/sun/javacard/samples/sigMsgRec/javacard/Method.cap
0x80 0xB2 0x07 0x00 0x00 0x7F;
0x80 0xB4 0x07 0x00 0x20 0x07 0x02 0x24 0x00 0x04 0x41 0x18 0x8C 0x00 0x1A 0x19 0x1E 0x25 0x29 0x04 0x16 0x04 0x61 0x08 0x18 0x8B 0x00 0x07 0x70 0x0C 0x18 0x19 0x1E 0x04 0x41 0x16 0x04 0x7F;
0x80 0xB4 0x07 0x00 0x20 0x8B 0x00 0x0C 0x18 0x07 0x11 0x02 0x00 0x03 0x8D 0x00 0x0F 0x94 0x00 0x00 0x10 0x87 0x00 0x18 0x08 0x11 0x02 0x00 0x03 0x8D 0x00 0x0F 0x94 0x00 0x00 0x06 0x87 0x7F;
0x80 0xB4 0x07 0x00 0x20 0x01 0xAD 0x01 0x7B 0x00 0x08 0x03 0x7B 0x00 0x08 0x92 0x8E 0x04 0x00 0x06 0x06 0xAD 0x01 0x7B 0x00 0x09 0x03 0x7B 0x00 0x09 0x92 0x8E 0x04 0x00 0x06 0x07 0xAD 0x7F;
0x80 0xB4 0x07 0x00 0x20 0x00 0x7B 0x00 0x0A 0x03 0x7B 0x00 0x0A 0x92 0x8E 0x04 0x00 0x10 0x06 0xAD 0x00 0x7B 0x00 0x09 0x03 0x7B 0x00 0x09 0x92 0x8E 0x04 0x00 0x10 0x07 0x18 0x10 0x42 0x7F;
0x80 0xB4 0x07 0x00 0x20 0x05 0x8D 0x00 0x0B 0x87 0x02 0x18 0x10 0x1E 0x03 0x8D 0x00 0x0D 0x94 0x00 0x00 0x0E 0x87 0x03 0x18 0x03 0x88 0x04 0x7A 0x05 0x30 0x8F 0x00 0x11 0x3D 0x18 0x1D 0x7F;
0x80 0xB4 0x07 0x00 0x20 0x1E 0x8C 0x00 0x12 0x3B 0x7A 0x02 0x21 0x19 0x8B 0x00 0x13 0x2D 0x1A 0x03 0x25 0x61 0x0A 0x1A 0x04 0x25 0x10 0xA4 0x6B 0x03 0x7A 0x1A 0x04 0x25 0x73 0x00 0x22 0x7F;
0x80 0xB4 0x07 0x00 0x20 0x00 0x10 0x00 0x12 0x00 0x0D 0x00 0x14 0x00 0x1B 0x18 0x19 0x8C 0x00 0x14 0x70 0x16 0x18 0x19 0x8C 0x00 0x15 0x70 0x0F 0x18 0x19 0x8C 0x00 0x16 0x70 0x08 0x11 0x7F;
0x80 0xB4 0x07 0x00 0x20 0x6D 0x00 0x8D 0x00 0x17 0x7A 0x09 0x23 0x19 0x8B 0x00 0x13 0x2D 0x19 0x8B 0x00 0x18 0x32 0x04 0x05 0x8D 0x00 0x19 0x28 0x04 0xAD 0x03 0xAD 0x01 0x04 0x8E 0x03 0x7F;
0x80 0xB4 0x07 0x00 0x20 0x00 0x0E 0x00 0x18 0xAD 0x03 0x1A 0x08 0x1F 0xAD 0x02 0x03 0x15 0x04 0x03 0x8E 0x08 0x00 0x0E 0x02 0x89 0x05 0xAD 0x02 0xAF 0x05 0x15 0x04 0x03 0x26 0x11 0xFF 0x7F;
0x80 0xB4 0x07 0x00 0x20 0x00 0x53 0x10 0x08 0x4F 0x5B 0x38 0xAD 0x02 0xAF 0x05 0x04 0x41 0x15 0x04 0x03 0x26 0x11 0x00 0xFF 0x53 0x5B 0x38 0x19 0x8B 0x00 0x1B 0x3B 0x19 0xAF 0x05 0x05 0x7F;
0x80 0xB4 0x07 0x00 0x20 0x41 0x8B 0x00 0x1C 0x19 0xAD 0x02 0x03 0xAF 0x05 0x05 0x41 0x8B 0x00 0x1D 0x7A 0x05 0x25 0xAD 0x03 0xAD 0x00 0x05 0x8E 0x03 0x00 0x0E 0x00 0x03 0x31 0x19 0x8B 0x7F;
0x80 0xB4 0x07 0x00 0x20 0x00 0x13 0x2E 0x1B 0x07 0x25 0x11 0x00 0xFF 0x53 0x29 0x04 0x19 0x8B 0x00 0x18 0x29 0x05 0x1B 0x08 0xAD 0x02 0x03 0x16 0x05 0x8D 0x00 0x1E 0x3B 0xAD 0x03 0xAD 0x7F;
0x80 0xB4 0x07 0x00 0x20 0x02 0x03 0x16 0x05 0x8E 0x04 0x00 0x0E 0x01 0x29 0x06 0xAD 0x03 0xAD 0x02 0x03 0x03 0x8E 0x04 0x00 0x0E 0x03 0x31 0x1E 0x61 0x08 0x11 0x91 0x02 0x8D 0x00 0x17 0x7F;
0x80 0xB4 0x07 0x00 0x20 0x7A 0x05 0x24 0x19 0x8B 0x00 0x13 0x2D 0x1A 0x07 0x25 0x11 0x00 0xFF 0x53 0x32 0x19 0x8B 0x00 0x18 0x29 0x04 0x1A 0x08 0xAD 0x02 0x03 0x16 0x04 0x8D 0x00 0x1E 0x7F;
0x80 0xB4 0x07 0x00 0x20 0x3B 0xAE 0x04 0x61 0x4B 0xAD 0x03 0xAD 0x00 0x05 0x8E 0x03 0x00 0x0E 0x00 0xAD 0x03 0xAD 0x02 0x03 0x16 0x04 0x8E 0x04 0x00 0x0E 0x01 0x29 0x05 0xAD 0x02 0x03 0x7F;
0x80 0xB4 0x07 0x00 0x20 0x16 0x05 0x11 0xFF 0x00 0x53 0x10 0x08 0x4F 0x5B 0x38 0xAD 0x02 0x04 0x16 0x05 0x11 0x00 0xFF 0x53 0x5B 0x38 0x19 0x8B 0x00 0x1B 0x3B 0x19 0x05 0x8B 0x00 0x1C 0x7F;
0x80 0xB4 0x07 0x00 0x20 0x19 0xAD 0x02 0x03 0x05 0x8B 0x00 0x1D 0x18 0x04 0x88 0x04 0x70 0x1A 0x18 0x03 0x88 0x04 0xAD 0x03 0xAD 0x02 0x03 0x16 0x04 0x8E 0x04 0x00 0x0E 0x03 0x61 0x08 0x7F;
0x80 0xB4 0x07 0x00 0x07 0x11 0x91 0x02 0x8D 0x00 0x17 0x7A 0x7F;
0x80 0xBC 0x07 0x00 0x00 0x7F;

// com/sun/javacard/samples/sigMsgRec/javacard/StaticField.cap
0x80 0xB2 0x08 0x00 0x00 0x7F;
0x80 0xB4 0x08 0x00 0x20 0x08 0x00 0x96 0x00 0x06 0x00 0x03 0x00 0x03 0x03 0x00 0x03 0x01 0x00 0x01 0x03 0x00 0x40 0xBE 0xDF 0xD3 0x7A 0x08 0xE2 0x9A 0x58 0x27 0x54 0x2A 0x49 0x18 0xCE 0x7F;
0x80 0xB4 0x08 0x00 0x20 0xE4 0x1A 0x60 0xDC 0x62 0x75 0xBD 0xB0 0x8D 0x15 0xA3 0x65 0xE6 0x7B 0xA9 0xDC 0x09 0x11 0x5F 0x9F 0xBF 0x29 0xE6 0xC2 0x82 0xC8 0x35 0x6B 0x0F 0x10 0x9B 0x19 0x7F;
0x80 0xB4 0x08 0x00 0x20 0x62 0xFD 0xBD 0x96 0x49 0x21 0xE4 0x22 0x08 0x08 0x80 0x6C 0xD1 0xDE 0xA6 0xD3 0xC3 0x8F 0x03 0x00 0x40 0x84 0x21 0xFE 0x0B 0xA4 0xCA 0xF9 0x7D 0xBC 0xFC 0x0E 0x7F;
0x80 0xB4 0x08 0x00 0x20 0xA9 0xBB 0x7A 0xBD 0x7D 0x65 0x40 0x2B 0x08 0xC6 0xDF 0xC9 0x4B 0x09 0x6A 0x29 0x3B 0xC2 0x42 0x88 0x23 0x44 0xAF 0x08 0x82 0x4C 0xFF 0x42 0xA4 0xB8 0xD2 0xDA 0x7F;
0x80 0xB4 0x08 0x00 0x19 0xCC 0xEE 0xC5 0x34 0xED 0x71 0x01 0xAB 0x3B 0x76 0xDE 0x6C 0xA2 0xCB 0x7C 0x38 0xB6 0x9A 0x4B 0x28 0x01 0x00 0x00 0x00 0x00 0x7F;
0x80 0xBC 0x08 0x00 0x00 0x7F;

// com/sun/javacard/samples/sigMsgRec/javacard/ConstantPool.cap
0x80 0xB2 0x05 0x00 0x00 0x7F;
0x80 0xB4 0x05 0x00 0x20 0x05 0x00 0x7E 0x00 0x1F 0x02 0x00 0x00 0x00 0x02 0x00 0x00 0x01 0x02 0x00 0x00 0x03 0x02 0x00 0x00 0x02 0x02 0x00 0x00 0x05 0x02 0x00 0x00 0x04 0x01 0x81 0x08 0x7F;
0x80 0xB4 0x05 0x00 0x20 0x00 0x03 0x80 0x03 0x01 0x05 0x00 0x00 0x04 0x05 0x00 0x00 0x02 0x05 0x00 0x00 0x00 0x06 0x80 0x08 0x0D 0x03 0x80 0x03 0x02 0x06 0x81 0x0F 0x00 0x01 0x81 0x19 0x7F;
0x80 0xB4 0x05 0x00 0x20 0x00 0x06 0x81 0x0D 0x00 0x01 0x81 0x09 0x00 0x01 0x00 0x00 0x00 0x06 0x00 0x00 0x01 0x03 0x80 0x0A 0x01 0x06 0x00 0x00 0xE3 0x06 0x00 0x01 0x4D 0x06 0x00 0x01 0x7F;
0x80 0xB4 0x05 0x00 0x20 0x9E 0x06 0x80 0x07 0x01 0x03 0x80 0x0A 0x06 0x06 0x80 0x08 0x0F 0x06 0x80 0x03 0x00 0x03 0x80 0x0A 0x07 0x03 0x80 0x0A 0x09 0x03 0x80 0x0A 0x05 0x06 0x80 0x10 0x7F;
0x80 0xB4 0x05 0x00 0x01 0x02 0x7F;
0x80 0xBC 0x05 0x00 0x00 0x7F;

// com/sun/javacard/samples/sigMsgRec/javacard/RefLocation.cap
0x80 0xB2 0x09 0x00 0x00 0x7F;
0x80 0xB4 0x09 0x00 0x20 0x09 0x00 0x63 0x00 0x29 0x2E 0x0F 0x02 0x0F 0x0F 0x0F 0x16 0x0D 0x04 0x64 0x02 0x09 0x05 0x0B 0x02 0x02 0x0F 0x02 0x14 0x08 0x03 0x0A 0x02 0x20 0x09 0x02 0x0C 0x7F;
0x80 0xB4 0x09 0x00 0x20 0x02 0x2B 0x09 0x04 0x02 0x08 0x02 0x0C 0x0E 0x16 0x09 0x06 0x02 0x02 0x00 0x36 0x05 0x0D 0x0C 0x09 0x04 0x0B 0x04 0x07 0x04 0x05 0x06 0x04 0x05 0x06 0x04 0x05 0x7F;
0x80 0xB4 0x09 0x00 0x20 0x06 0x04 0x05 0x08 0x09 0x04 0x0C 0x07 0x08 0x23 0x07 0x07 0x08 0x07 0x05 0x06 0x0B 0x11 0x28 0x09 0x0B 0x0C 0x07 0x0E 0x0C 0x0C 0x0D 0x0B 0x07 0x0D 0x0C 0x0E 0x7F;
0x80 0xB4 0x09 0x00 0x06 0x0C 0x20 0x06 0x08 0x15 0x09 0x7F;
0x80 0xBC 0x09 0x00 0x00 0x7F;

0x80 0xBA 0x00 0x00 0x00 0x7F;
// resume printing output during functional test
output on;

//create applet With AID 0xA0 0x00 0x00 0x00 0x62 0x03 0x01 0x0C 0x0C 0x01
echo "creating applet";
//0x80 0xB8 0x00 0x00 0x0D 0x0B 0xA0 0x00 0x00 0x00 0x62 0x03 0x01 0x0C 0x07 0x07 0x01 0x00 0x7F;
0x80 0xB8 0x00 0x00 0x0C 0x0A 0xA0 0x00 0x00 0x00 0x62 0x03 0x01 0x0C 0x0C 0x01 0x00 0x7F;

// select applet         0xA0 0x00 0x00 0x00 0x62 0x03 0x01 0x0C 0x0C 0x01
0x00 0xA4 0x04 0x00 0x0A 0xA0 0x00 0x00 0x00 0x62 0x03 0x01 0x0C 0x0C 0x01 0x7F;
//expected 90 00

0x80 0x10 0x00 0x00 0x01 0x01 0x7F;
//Sign expected the actual data

//0x80 0x11 0x00 0x00 0x40 0x41 0xa3 0x79 0x00 0xbd 0xf3 0xd8 0x3c 0xca 0x13 0xd1 0xa9 0x34 0xd7 0xb5 0xcc 0x8e 0xed 0x36 0xa3 0xbc 0x57 0x3e 0xb2 0xb6 0xec 0x77 0xb0 0xcb 0x33 0x4d 0x43 0x1a 0x3f 0x25 0x82 0x64 0x80 0xb1 0x2f 0x98 0xdc 0xa9 0xd8 0x44 0xe9 0x92 0x2c 0x42 0x51 0x9d 0x8c 0x97 0x3e 0x97 0xcd 0x97 0x98 0xd0 0xf0 0x93 0xbb 0x93 0xf3 0x7F;
//Verify expected 90 00

0x80 0x11 0x00 0x00 0x40 0xa3 0x49 0x1d 0x51 0x55 0x05 0x49 0x71 0xba 0xdc 0x77 0x22 0xce 0x9a 0x51 0x71 0xf8 0xb1 0x88 0x8d 0x55 0x05 0xd5 0x2b 0xae 0xf6 0xb7 0x04 0xd9 0x1d 0x09 0x35 0x17 0xec 0x73 0x11 0xd5 0x7f 0xfd 0xeb 0xb3 0xd9 0x98 0x45 0xf7 0x8a 0xb6 0x72 0x21 0x44 0xa1 0x32 0xb3 0xa1 0xce 0x72 0xc5 0x6d 0xcc 0xee 0x18 0x64 0x2e 0x76 0x7F;

powerdown;

powerdown;

