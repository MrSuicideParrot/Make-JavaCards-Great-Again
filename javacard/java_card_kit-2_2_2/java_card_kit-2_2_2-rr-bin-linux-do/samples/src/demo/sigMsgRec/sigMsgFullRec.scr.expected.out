CLA: 00, INS: a4, P1: 04, P2: 00, Lc: 09, a0, 00, 00, 00, 62, 03, 01, 08, 01, Le: 00, SW1: 90, SW2: 00
CAP file download section. Output suppressed.
OUTPUT OFF;
OUTPUT ON;
creating applet
CLA: 80, INS: b8, P1: 00, P2: 00, Lc: 0c, 0a, a0, 00, 00, 00, 62, 03, 01, 0c, 0c, 01, 00, Le: 0a, a0, 00, 00, 00, 62, 03, 01, 0c, 0c, 01, SW1: 90, SW2: 00
CLA: 00, INS: a4, P1: 04, P2: 00, Lc: 0a, a0, 00, 00, 00, 62, 03, 01, 0c, 0c, 01, Le: 00, SW1: 90, SW2: 00
CLA: 80, INS: 10, P1: 00, P2: 00, Lc: 01, 01, Le: 42, a3, 49, 1d, 51, 55, 05, 49, 71, ba, dc, 77, 22, ce, 9a, 51, 71, f8, b1, 88, 8d, 55, 05, d5, 2b, ae, f6, b7, 04, d9, 1d, 09, 35, 17, ec, 73, 11, d5, 7f, fd, eb, b3, d9, 98, 45, f7, 8a, b6, 72, 21, 44, a1, 32, b3, a1, ce, 72, c5, 6d, cc, ee, 18, 64, 2e, 76, 00, 01, SW1: 90, SW2: 00
CLA: 80, INS: 11, P1: 00, P2: 00, Lc: 40, a3, 49, 1d, 51, 55, 05, 49, 71, ba, dc, 77, 22, ce, 9a, 51, 71, f8, b1, 88, 8d, 55, 05, d5, 2b, ae, f6, b7, 04, d9, 1d, 09, 35, 17, ec, 73, 11, d5, 7f, fd, eb, b3, d9, 98, 45, f7, 8a, b6, 72, 21, 44, a1, 32, b3, a1, ce, 72, c5, 6d, cc, ee, 18, 64, 2e, 76, Le: 00, SW1: 90, SW2: 00
